Phaethon/Tests/IntegrationTests/LoginTest.cs

```diff
@@ -0,0 +1,202 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Data.Entity;
+using System.Data.Entity.Core.Objects;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Runtime.ExceptionServices;
+using System.Text;
+using System.Threading.Tasks;
+using Core.Model;
+using InternalApi.DataAccess;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using System.Security.Cryptography;
+using InternalApi.DataManagement;
+
+namespace Tests.IntegrationTests
+{
+
+   public class LoginTest : InternalTestFakeServerBase
+    {
+        private Login userModel;
+        private Login userModel2;
+        private int count1;
+        private int count2;
+        private string name;
+        private string name2;
+        private string OGPass;
+        private LoginDa loginDa;
+
```

Comment:
https://github.com/RaidenRabit/Phaethon/blob/da6d6b8edb567c6e375539f3d112132bbce69993/Phaethon/Tests/IntegrationTests/LoginTest.cs#L26-L30

These should be local variables, as they are used only in 1 test.